app.json

```diff
@@ -10,7 +10,7 @@
     },
     "POSTGREST_VER": {
       "description": "Version of PostgREST to deploy",
-      "value": "8.0.0"
+      "value": "9.0.0"
     },
     "DB_URI": {
```

Comment:
Can't comment on the lines below - but there are a lot of config options / environment variables in here. We had plenty of changes to those, that are not reflected here - already between v7 and 8.

Should we update all of those here? Does it matter?

Reply:
Hm, right. I think I'll remove this change. A postgrest-heroku user could do it later.

(Ideally we'd move this into postgrest-heroku but IIRC that can't be done)

Reply:
Let's discuss in #2049 how we're going to deal with this.